linux/dir_tools.bsh

```diff
@@ -470,4 +471,65 @@ function relative_path()
   else
     echo "${ans[*]}"
   fi
-}
\ No newline at end of file
+}
+
+#**
+# .. function:: strip_copy
+#
+# :Arguments: * ``$1`` - The source directory
+#             * ``$2`` - The destination directory
+#             * ``$3`` - Strip this many leading components from file names on ``cp``
+#
+# Replicates the behavior of ``tar --strip-components``, but when applied to a ``cp`` scenario.
+#
+# .. note:
+#
+#    ``cp`` will process files in a different order than ``tar --strip-components``. In case there are multiple files with the same name at a stripped depth,files may be overwritten differently than ``tar --strip-components`` would have. While ``tar`` keeps the first one in the archive order, ``cp`` will keep the last one in ``cp`` order. And as both order and behaviors are different, the result could be the same, or different.
```

Comment:
```suggestion
#    ``cp`` will process files in a different order than ``tar --strip-components``. In the case where there are multiple files with the same name at a stripped depth, the file may be overwritten in a different order than ``tar --strip-components`` would have. While ``tar`` keeps the first one in the archive order, ``cp`` will keep the last one in ``cp`` order. Consequently, the result will likely be different.
```

Reply:
```suggestion
#    ``cp`` will process files in a different order than ``tar --strip-components``. In case there are multiple files with the same name at a stripped depth,files may be overwritten differently than ``tar --strip-components`` would have. While ``tar`` keeps the first one in the archive order, ``cp`` will keep the last one in ``cp`` order. Consequently, the order is not guaranteed as both order and behaviors are different.
```